crypto/sessionkey_streaming_test.go

```diff
@@ -0,0 +1,89 @@
+package crypto
+
+import (
+	"bytes"
+	"io"
+	"testing"
+
+	"github.com/pkg/errors"
+)
+
+func TestSessionKey_EncryptStream(t *testing.T) {
```

Comment:
Same here, add some test that ensure compatibility with non-streaming API

Reply:
Done